Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?= -n -j "auto"
```

Comment:
Logs output to a text file so it's easier to track what has gone wrong. Already excluded from git.